src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/ratio-tap-changer-pane/ratio-tap-changer-pane.js

```diff
@@ -275,61 +270,60 @@ const RatioTapChangerPane = ({
-                            {regulationModeField}
-                        </Grid>
-                        {watchLRegulationMode && (
-                            <>
-                                <Grid item xs={4}>
-                                    {targetVoltage1Field}
-                                </Grid>
-                                <Grid item xs={4}>
-                                    {targetDeadbandField}
-                                </Grid>
-                            </>
-                        )}
-                    </Grid>
-                )}
-                {isRatioTapLoadTapChangingCapabilitiesOn && (
-                    <Grid item container spacing={2}>
-                        <Grid
-                            item
-                            xs={4}
-                            style={{
-                                display: 'flex',
-                                justifyContent: 'flex-end',
-                                alignItems: 'center',
-                            }}
-                        >
-                            <FormattedMessage
-                                id="RegulatedTerminal"
-                                disabled={true}
-                            />
+                    <>
+                        <Grid item container spacing={2}>
+                            <Grid item xs={4}>
+                                {regulationModeField}
+                            </Grid>
+                            {regulationModeWatch && (
+                                <>
+                                    <Grid item xs={4}>
+                                        {targetVoltage1Field}
+                                    </Grid>
+                                    <Grid item xs={4}>
+                                        {targetDeadbandField}
+                                    </Grid>
+                                </>
+                            )}
                         </Grid>
-                        <Grid item xs={4}>
-                            {regulationTypeField}
+                        <Grid item container spacing={2}>
+                            <Grid
+                                item
+                                xs={4}
+                                style={{
+                                    display: 'flex',
+                                    justifyContent: 'flex-end',
+                                    alignItems: 'center',
+                                }}
+                            >
+                                <FormattedMessage
+                                    id="RegulatedTerminal"
+                                    disabled={true}
+                                />
+                            </Grid>
+                            <Grid item xs={4}>
+                                {regulationTypeField}
+                            </Grid>
+                            {isRatioTapLoadTapChangingCapabilitiesOn &&
```

Comment:
You can remove this one which is useless

Reply:
Done